src/server/bin/plugin/pluginRemover.js

```diff
@@ -0,0 +1,25 @@
+﻿var fs = require('fs');
+var rimraf = require('rimraf');
+
+module.exports = {
+  remove: remove
+};
+
+function remove(settings, logger) {
+  try {
+    try {
+      fs.statSync(settings.pluginPath);
+    }
+    catch (e) {
+      logger.log('Plugin ' + settings.package + ' does not exist.');
+      return;
+    }
+
+    logger.log('Removing ' + settings.package + '...');
+
+    rimraf.sync(settings.pluginPath);
+  } catch (ex) {
+    logger.error(ex.message);
```

Comment:
Can we log something more informative here, maybe:
 `Unable to remove plugin "${settings.package}" because of error: "${ex.message}"`